### ISSUE TITLE
Pete Buttigieg not in the presidential candidate list when viewing 2020 two-year time period

### ISSUE THREAD

**Issue by patphongs** [fecgov/fec-cms#3917]
### Summary
Pete Buttigieg does not appear in the presidential candidate list when looking at the 2020 two year time period. The candidate only appears when viewing the full 2020 presidential election year.

### 2020 two year period only
**Frontend datatable:**
https://www.fec.gov/data/candidates/president/?q=P00010298&election_year=2020&cycle=2020&election_full=false

**API call:**
https://api.open.fec.gov/v1/candidates/totals/?api_key=DEMO_KEY&sort_hide_null=false&sort_nulls_last=true&q=P00010298&election_year=2020&cycle=2020&election_full=false&sort=-receipts&per_page=30&page=1&office=P

![Screen Shot 2020-07-15 at 9 58 10 AM](https://user-images.githubusercontent.com/12799132/87554213-d8e21f00-c681-11ea-92da-2c85d370839d.png)

### Full election cycle
**Frontend datatable:** 
https://www.fec.gov/data/candidates/president/?q=P00010298&election_year=2020&cycle=2020&election_full=true

**API call:**
https://api.open.fec.gov/v1/candidates/totals/?api_key=DEMO_KEY&sort_hide_null=false&sort_nulls_last=true&q=P00010298&election_year=2020&cycle=2020&election_full=true&sort=-receipts&per_page=30&page=1&office=P

![Screen Shot 2020-07-15 at 9 58 33 AM](https://user-images.githubusercontent.com/12799132/87554176-cd8ef380-c681-11ea-927e-5644bbf6e28e.png)

### Completion criteria
- [ ] query with `election_full=false`  returns results for the 2-year period specified

**Comment by JonellaCulmer** [comment]
I do not believe this work requires usability testing. From previous testing with our users, I know for a fact that when an individual is researching a candidate they know to have run during a particular time period, the expectation is to find that candidate there. 

If we are unable to provide an accurate list of candidates because our data will not allow it, we need to provide clear information in a methodology as to why that's the case.

**Comment by djgarr** [comment]
Can this issue be closed?

**Comment by PaulClark2** [comment]
@djgarr yes this issue can be closed.

**Comment by djgarr** [comment]
Thanks @PaulClark2 !
